app/assets/javascripts/routes/reviews_index.js

```diff
@@ -0,0 +1,5 @@
+App.ReviewsIndexRoute = App.AuthenticatedRoute.extend({
+  model: function() {
+    return this.modelFor('course').get('reviews');
```

Comment:
1 seems like bad UX unless there's a "hey did you fatfinger this or do you really wanna leave without saving" warning. 2 seems confusing, since it's not clear whether the half-finished review was persisted or not from UI alone. 3 would probably be easy to implement, right? Just only show persisted reviews instead of all reviews. 3's easiest.